frontend/src/app/modules/exclusion-dates/exclusion-dates-picker/exclusion-dates-picker.component.ts

```diff
@@ -30,6 +31,12 @@ export class ExclusionDatesPickerComponent extends BaseComponent implements OnIn
 
     ngOnInit() {
         this.formGroup = this.getDefaultFormGroup();
+        const startDelayMilliseconds = (60 - new Date().getSeconds()) * 1000;
+        const minuteMilliseconds = 60000;
+
+        timer(startDelayMilliseconds, minuteMilliseconds)
+            .pipe(this.untilThis)
+            .subscribe(() => this.updateStartRangeValidators());
```

Comment:
Why do we need this?

Reply:
The reason is that when user enters time range and he selected current date as exclusion date, then all time ranges which start before current moment of time are considered to be invalid. This peace of code makes sure that all entered, but not saved time ranges are valid in current moment of time

Reply:
But why is this time range invalid?

Reply:
Fixed